support/support-bundle.yaml

```diff
@@ -61,6 +61,11 @@ spec:
       command: ["/bin/sh"]
       args: ["-c","df -h /bitnami/rabbitmq/mnesia"]
       timeout: 10s
+  hostCollectors:
+  - run:
+      collectorName: "helm-list"
+      command: "/bin/sh"
+      args: ["-c", "helm list --all-namespaces" ]
```

Comment:
I hope the output from this in a multitenant environment doesn't overwhelm us 👀

Reply:
yea, the host collectors don't seem to respect the namespace parameter that the other collectors do.  but think the list of install charts, even in a busy environment, will be small compared to the other data we are getting